assignments/final_demo.md

```diff
@@ -143,18 +78,15 @@ This will help keep things in one consolidated location and will be used to help
  
 | Area | Description| Weight |
 | --- | --- | --- |
-| Problem and Solution | Gives a concise explanation of the problem and solution | 5.0 |
-| Progress | Has clearly progressed from Assignment 3 and addressed any feedback`*` | 25.0 |
-| Vision | Shows a better understanding of the problem from A3, explains the vision of the product, does not leave questions unanswered`**` | 10.0 |
-| Software | Has working code that runs and delivers a demo. Does not need to use command line (unless that is the main product UI!), code, unit tests, or other mockups to show the main flow  | 30.0 |
-| Reflection | Thoughtfully reflects on the build plan and problems faced. Outlines plans into the future. | 15.0 |
-| Presentation | Engaging presentation, ensures only relevant parts are shown and does not sugar-coat | 15.0 |
+| Problem and Solution | Gives a concise explanation of the problem and solution. | 5.0 |
+| Product Vision | Explains the vision of the product, tells us about the future of the product, does not leave questions unanswered`**` | 15.0 |
+| Demo | The demonstration covers the main flows of the application. The flows seem to address the problem and be a solution. | 25.0 |
+| Software | Has working code that runs and delivers a demo. Does not need to use command line (unless that is the main product UI!), code, unit tests, or other mockups to show the main flow. Mockups can be used to show non-main-path or future plans.  | 25.0 |
+| Reflection | Thoughtfully reflects on the build plan, architecture diagram, product, and problems faced. Describes how these changed over the term. Outlines plans into the future. | 15.0 |
```

Comment:
It would be really great to see the students incorporate this into some parts of their demo as well I think. For example, having them go back to certain parts of the demo and talk about future extensions and potentially what about that aspect was challenging for them. That way there's a visual in the product attached to their reflections